be/src/exprs/string_functions.cpp

```diff
@@ -2788,6 +2788,96 @@ StatusOr<ColumnPtr> StringFunctions::strcmp(FunctionContext* context, const Colu
     return VectorizedStrictBinaryFunction<strcmpImpl>::evaluate<TYPE_VARCHAR, TYPE_INT>(columns[0], columns[1]);
 }
 
+// strpos without instance parameter
+StatusOr<ColumnPtr> StringFunctions::strpos(FunctionContext* context, const Columns& columns) {
+    RETURN_IF_COLUMNS_ONLY_NULL(columns);
+
+    const ColumnPtr& haystack = columns[0];
+    const ColumnPtr& needle = columns[1];
+    ColumnPtr instance = ColumnHelper::create_const_column<TYPE_INT>(1, columns[0]->size());
```

Comment:
TODO: no need to create an extra `instance` for performance issue, use template or function parameter seems fine.